src/gax/src/error/rpc/generated/mod.rs

```diff
@@ -335,21 +339,27 @@ pub mod bad_request {
         ///
         /// Consider the following:
         ///
-        ///     message CreateContactRequest {
-        ///       message EmailAddress {
-        ///         enum Type {
-        ///           TYPE_UNSPECIFIED = 0;
-        ///           HOME = 1;
-        ///           WORK = 2;
-        ///         }
+        /// ```norust
+        /// message CreateContactRequest {
+        ///   message EmailAddress {
+        ///     enum Type {
+        ///       TYPE_UNSPECIFIED = 0;
+        ///       HOME = 1;
+        ///       WORK = 2;
+        ///     }
+        /// ```
```

Comment:
As noted elsewhere, ideally these two blockquotes would be merged, but the code (maybe by accident) is doing the right thing:

https://spec.commonmark.org/0.31.2/#example-242

The comments upstream (in the protos) need fixing.